### ISSUE TITLE
Run menu re-org script on prod and fix Lovell

### ISSUE THREAD

**Issue by swirtSJW** [department-of-veterans-affairs/va.gov-cms#17202]
## User Story or Problem Statement

Relates to https://github.com/department-of-veterans-affairs/va.gov-cms/issues/16233 

We need to execute the script introduced by [this PR](https://github.com/department-of-veterans-affairs/va.gov-cms/pull/17189) once it lands on prod.

## Notes

- It does NOT have to be urgent and go right after the code deployment.  It can wait until whenever we choose.  
- The script takes roughly an hour to run, but there is no risk window, There is nothing that can happen during that hour that we need to undo or re-do.

### Devops
Once the code from https://github.com/department-of-veterans-affairs/va.gov-cms/pull/17189 lands on prod:
- [ ] SSH to prod
- [ ] become the drush user
- [ ] Set the state to the last menu that was processed correctly `ddev drush state:set script_library__va_gov_vamc_get_system_menus 117`
- [ ] re-run `drush scr ./scripts/content/VACMS-16233-reorder-vamc-menu-items.php`
   - will count off like this
![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/5752113/25cb7744-db91-4308-9db0-a762607d9012)

   - It should take somewhere just 15 min to run. 
   - If for some reason it fails, it is safe to run as many times as needed to finish. It will pick up where it left off.
- [ ] Notify Facilities team that it  is done.

### Facilities team 
Once the devops process ^^ is complete:
  - [x] Login as admin
  - [x] Go to the [Lovell menu](https://prod.cms.va.gov/admin/structure/menu/manage/lovell-federal-health-care) 
  - Lovell has duplicate menu items due to its dual nature.
  - One of each of the pairs will be in the right order, the others will not be. (and some may not have pairs)
  - [x] Move the errant menu items so they are adjacent to their twin and in the order matching
```
About us
Programs
Research
Policies
VA police
Work with us
Contact us
```
  - [x] Save the menu.

## Team
Please check the team(s) that will do this work.

- [x] CMS Team
- [ ] Public Websites
- [x] Facilities
- [ ] Accelerated Publishing

**Comment by swirtSJW** [comment]
I just updated the steps to have this pick up where it left off, once https://github.com/department-of-veterans-affairs/va.gov-cms/pull/17236 lands on prod.

**Comment by omahane** [comment]
I've moved Lovell.

**Comment by michelle-dooley** [comment]
@7hunderbird can assist as needed.

**Comment by jilladams** [comment]
@xiongjaneg @omahane this ticket shows as a Facilities team ticket in our Icebox, but also in Sprint 104. Could one of you confirm if this is in current sprint?

**Comment by xiongjaneg** [comment]
@jilladams This was a script we needed help from CMS Team to run so I've updated the labels.

**Comment by gracekretschmer-metrostar** [comment]
@edmund-dunn

**Comment by jilladams** [comment]
@edmund-dunn @gracekretschmer-metrostar could you clarify if there is work remaining on this ticket in upcoming sprint, or if it's been run in prod and we can verify our changes?

**Comment by omahane** [comment]
@gracekretschmer-metrostar I asked @edmund-dunn about this yesterday, and [he said it ran successfully](https://dsva.slack.com/archives/CT4GZBM8F/p1709067694970729?thread_ts=1708010348.181899&cid=CT4GZBM8F).

**Comment by gracekretschmer-metrostar** [comment]
Great! Disregard my comment then @omahane. I don't know why there are two tickets.

**Comment by xiongjaneg** [comment]
Noted earlier comment that a separate PR for DevOps ran successfully. This ticket for Drupal work is complete. Verified menu order of Lovell on prod.
<img width="362" alt="Screenshot 2024-03-06 at 10 01 39 AM" src="https://github.com/department-of-veterans-affairs/va.gov-cms/assets/73648238/c1968c68-45e4-4d84-abcd-35434ad124ef">
<img width="362" alt="Screenshot 2024-03-06 at 10 01 52 AM" src="https://github.com/department-of-veterans-affairs/va.gov-cms/assets/73648238/89b30569-fbc1-4262-a3ad-c5a1fea684c1">
<img width="312" alt="Screenshot 2024-03-06 at 10 02 11 AM" src="https://github.com/department-of-veterans-affairs/va.gov-cms/assets/73648238/43aae208-205a-4b22-b66b-5c608d385aba">
